src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -153,7 +156,7 @@ public TagHelperDescriptorFactory(bool designTime)
                         typeName,
                         assemblyName,
                         attributeDescriptors,
-                        requiredAttributes: Enumerable.Empty<string>(),
+                        requiredAttributes: Enumerable.Empty<TagHelperRequiredAttributeDescriptor>(),
```

Comment:
Yes it will. However, tooling already has to react to a complete re-write of the tool so this is ok.